src/stylesheets/elements/_inputs.scss

```diff
@@ -77,8 +77,19 @@ select {
   label {
     margin-top: 0;
   }
+
+  .usa-input-inline {
+    border: $input-border-width solid $color-gray;
+    width: inherit;
+  }
+
+  .usa-input-inline-error {
+    border: 3px solid $color-secondary-dark;
```

Comment:
@maya can the code above be placed in with the date input guidance on standards.usa.gov?  I experienced similar behavior and shouldn't have to hunt all the way down here to find out how to implement error styling.